tests/testdata/formbook_a17cc6bd4e2c3bde5c4034cdb150fc594bd408b877dc056f43eaefbd44e59107.txt

```diff
@@ -0,0 +1,9 @@
+Key unknown:9f8c4d65fe000cdc9982b3e8c619ad51d87d577f
+Key unknown:793ae83f8584ff595adac5f30a938245bacd2b80
+Key unknown:3e0113f76d5a9bef1cc7251b794ef4c673b5020f
+Key unknown:d1d89a042a3b6cd903e129dd8bce0f619c0763a7
+Key unknown:b2f4cba568d55e85356603d0ebc1b993337c1426
+Key unknown:ae9cbfc2e8990a5af050b817a0c40f37152fb4db
+Key unknown:ce67a5fe27bcc291f30eeafe37dceee5e8d908a8
+NetLoc http://www.yourherogarden.net/dn7r/
+NetLoc japxo.online
```

Comment:
This netloc `japxo.online` (according to our own config) is listed both as a decoy domain and in the `domains` field. I assume if a domain is listed both in `domains` and `decoy_domains` then it's a decoy doimain after all? I can't check the module code right now, still missing some access.

If that's true, maybe we should handle this case and exclude all domains that are included in `decoy_domains` at any time? LGTM if I'm wrong and this domain is indeed malicious.